Meta/Lagom/CMakeLists.txt

```diff
@@ -384,10 +384,18 @@ if (BUILD_LAGOM)
 
     # GL
     file(GLOB LIBGL_SOURCES CONFIGURE_DEPENDS "../../Userland/Libraries/LibGL/*.cpp")
+    file(GLOB LIBGL_SHADER_SOURCES CONFIGURE_DEPENDS "../../Userland/Libraries/LibGL/Shaders/*.cpp")
     file(GLOB LIBGL_TEX_SOURCES CONFIGURE_DEPENDS "../../Userland/Libraries/LibGL/Tex/*.cpp")
+    file(GLOB LIBGL_SHADER_SOURCES CONFIGURE_DEPENDS "../../Userland/Libraries/LibGL/Shaders/*.cpp")
```

Comment:
This repeat from line 387 seems unnecesary

Reply:
Indeed. Must have sneaked in while rebasing.